notes/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from uuid import uuid4
+
+# Create your models here.
+
+
+class Note(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid4, editable= False)
+    title = models.CharField(max_length=200)
+    budkind= models.CharField(max_length=200)
+    description= models.TextField(blank=True)
+    favorite= models.BooleanField(default=False)
```

Comment:
do you know why we have class Note(models.Model)? it is because, Note is inheriting from models,Model. When we want to extend Note class, all we need to do is pass Note to the new class we created.